src/react-components/ui-root.js

```diff
@@ -2136,6 +2141,7 @@ class UIRoot extends Component {
 
                 {!streaming && (
                   <button
+                    aria-label="Favorite"
```

Comment:
Should this be "Toggle Favorited" as well?

Reply:
Yes